src/Message/Headers.php

```diff
@@ -58,6 +58,8 @@ private function parseHeader($key, $value)
                 return $this->decodeEmailAddress(current($value));
             case 'to':
                 // no break
+            case 'reply_to':
```

Comment:
so the code works, but it duplicates the code for from so...

```php
            case 'from':
                // no break
            case 'reply_to':
                return $this->decodeEmailAddress(current($value));
            case 'to':
                // no break

```

might be a cleaner way to express